### ISSUE TITLE
add anti forensics bash script

### ISSUE THREAD

**Issue by ghost** [piratelinux/Pirate-Linux#2]
https://github.com/int0x80/anti-forensics
Add support cryptsetup and dm-crypt planed?
my overlay https://github.com/m0fx64/amorse-overlay 
cryptsetup 
https://github.com/m0fx64/amorse-overlay/blob/master/sys-fs/cryptsetup/files/cryptsetup-1.6.2-nuke_keys.patch
http://www.kali.org/how-to/nuke-kali-linux-luks/
truecrypt patch https://github.com/m0fx64/amorse-overlay/blob/master/app-crypt/truecrypt/files/truecrypt-7.1a-goobypls.patch 
whitelist usb https://github.com/t0t3m/Porcupine and bash script support udev https://github.com/int0x80/anti-forensics/blob/master/shoopwhoop 

Youtube - "Anti-Forensic for the Louise" http://www.youtube.com/watch?v=-HK1JHR7LIM
Youtube - "Moar Anti-Forensic for the Louise" http://www.youtube.com/watch?v=i3nLrJrkYOc

can make the overlay?

**Comment by piratelinux** [comment]
Yes dm-crypt is planned for the full installer. Do you know how to write ncurses windows? I will need this for the installer, and also to ask for the user's encryption password. This nuking script looks interesting. I may implement it once I have the basics working.

**Comment by ghost** [comment]
I think encryption passvord not the best option, encryption usb token - key-file
https://github.com/m0fx64/amorse/blob/master/build#L71
I can not write ncurses windows

**Comment by piratelinux** [comment]
I can support both options. But I don't think keyfile is always the best option. Some people may not have an extra usb stick, and also, if you travel, your usb device may be searched. With a password, you can write it (part or all) down on paper and keep it in your pocket if you're travelling.
